angular/lib/src/core/linker/app_view.dart

```diff
@@ -584,7 +583,7 @@ abstract class AppView<T> {
     domRootRendererIsDirty = true;
   }
 
-  VoidFunc1<E> eventHandler0<E>(VoidFunc0 handler) {
+  void Function(E) eventHandler0<E>(void Function() handler) {
```

Comment:
Are we decided on what syntax to use when declaring function parameters? Both are valid:
```dart
void Function(E) eventHandler0<E>(void Function() handler) {}

void Function(E) eventHandler0<E>(void handler()) {}
```
I'm in favor of the latter as it's more concise, but I can understand the desire to be consistent with the former.